resources/views/streeplijst/overzicht.blade.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * @var $streeplijstoverzicht CsrDelft\entity\Streeplijst[]
+ */
+?>
+
+@extends('layout')
+
+@section('titel', 'Streeplijstgenerator')
+
+@section('content')
+	<h2>Streeplijstgenerator</h2>
+
+	<table class="table table-striped">
+		<thead>
+		<tr>
+			<th scope="col">Aanmaakdatum</th>
+			<th scope="col">Streeplijst</th>
+			<th scope="col">Leden van de streeplijst</th>
+			<th scope="col">Inhoud van de streeplijst</th>
+			<th scope="col"></th>
+		</tr>
+		</thead>
+		<tbody>
+		@forelse($streeplijstoverzicht as $streeplijst)
+			<tr>
+				<td>{{date_format_intl($streeplijst->aanmaakdatum, DATE_FORMAT)}}</td>
+				<td style="font-weight:bold">{{$streeplijst-> naam_streeplijst}}</td>
+				<td>{{truncate($streeplijst-> leden_streeplijst,318, '...')}}</td>
+				<td>{{$streeplijst-> inhoud_streeplijst}}</td>
+				<td>
+					<div class="btn-group" role="group" aria-label="Basic example">
+						<a href="/streeplijst/genereren/{{$streeplijst->id}}">
+						<span type="button"
+									class="btn btn-success btn-rounded btn-sm">Genereer</span></a>
+						<a href="/streeplijst/bewerken/{{$streeplijst->id}}">
+						<span type="button"
+									class="btn btn-warning btn-rounded btn-sm">Bewerk</span></a>
+						<a href="/streeplijst/verwijderen/{{$streeplijst->id}}">
+						<span type="button"
+									class="btn btn-danger btn-rounded btn-sm">Verwijder</span> </a>
+					</div>
+				</td>
+			</tr>
+		@empty
+			<tr>
+				<td colspan="5">Er zijn geen streeplijsten.</td>
+			</tr>
+		@endforelse
+		</tbody>
+	</table>
+
+	<h4>Maak of bewerk een lijst</h4>
+	<p>Een streeplijst heeft dat wat gestreept moet worden in de titelrij en heeft de leden op wie gestreept wordt in de
+		eerste kolomrij.</p>
+	<h4>Streeplijstselectie</h4>
+	<form action="/streeplijst/selectie" method="post">
+		{{printCsrfField()}}
+
+		<h5> Leden op de streeplijst</h5>
+		<div>
+			<em>Verticale:</em>
+			<input type="radio" name="verticale" id="alleverticale" value="alle" checked>
+			<label for="alleverticale"> Alle </label>
+			@foreach($verticalen as $verticale)
+				<input type="radio" name="verticale" id="{{$verticale->letter}}" value="{{$verticale->letter}}">
+				<label for="{{$verticale->letter}}"> {{$verticale->naam}}</label>
+			@endforeach
+		</div>
+
+		<div><em>Lichting:</em>
+			<input type="radio" name="lichting" id="alle" value="alle" checked>
+			<label for="alle"> Alle </label>
+			@foreach(range($jongstelidjaar-8, $jongstelidjaar) as $lichting)
+				<input type="radio" name="lichting" id="{{$lichting}}" value="{{$lichting}}">
+				<label for="{{$lichting}}"> {{$lichting}}</label>
+			@endforeach
+		</div>
+		<div><em>Type leden:</em>
```

Comment:
Beter een H4 ofzo of h6, een header is hier denk ik logischer

Reply:
Nee, dis mooier